Services/AuthenticationService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AppBackendTraining.Clients.Pms;
+using AppBackendTraining.Helpers;
+using AppBackendTraining.Models.Auth;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AppBackendTraining.Services
+{
+    public interface IAuthService
+    {
+        public Task<TokenView> Authenticate(AuthenticationRequest authenticateRequest);
+    }
+
+    public class AuthenticationService : IAuthService
+    {
+
+        private readonly IPmsService _pmsService;
+        private readonly AppSettings _appSettings;
+
+        public AuthenticationService(IPmsService pmsService, IOptions<AppSettings> appSettings)
+        {
+            _pmsService = pmsService;
+            _appSettings = appSettings.Value;
+        }
+
+        public async Task<TokenView> Authenticate(AuthenticationRequest authenticateRequest)
+        {
+            var pmsAccount = await _pmsService.GetAccountByEmail(authenticateRequest.Email);
```

Comment:
REM: 
When you don't need continuation on the same thread, you should rather use:

`
  var pmsAccount = await _pmsService.GetAccountByEmail(authenticateRequest.Email)
       .ConfigureAwait(continueOnCapturedContext: false)
`

This helps reduce contention. Usually only operations working with EF or request contexts needs to be run on same thread.